process.lua

```diff
@@ -1119,6 +1119,20 @@ function admin_level_valid(admin_level, is_unset_valid)
 	return (is_unset_valid and admin_level == "") or admin_level == "2" or admin_level == "3" or admin_level == "4"
 end
 
+-- Add boundary labels
+function relation_function(relation)
+	local boundary = Find("boundary")
+	local admin_level = Find("admin_level")
+	local name = Find("name")
+
+	if boundary == "administrative" and (admin_level == "2" or admin_level == "3" or admin_level == "4") then
+		LayerAsCentroid("boundary_labels")
+		MinZoom(2)
+		setNameAttributes()
+		Attribute("admin_level", admin_level)
```

Comment:
`admin_level` [is expected](https://shortbread-tiles.org/schema/1.0/#layer-boundary_labels) to be integer. Please use `AttributeNumeric` and the appropriate conversion function instead.

Reply:
Thanks